straxen/corrections_services.py

```diff
@@ -254,3 +254,21 @@ def cacheable_naming(*args, fmt='.npy', base='./resource_cache/'):
 class GainsNotFoundError(Exception):
     """Fatal error if a None value is returned by the corrections"""
     pass
+
+
+@strax.Context.add_method
+def apply_cmt_version(context, cmt_version):
```

Comment:
Although the shape of the options makes it already quite clear that these are CMT options, we could in principle agree on some semantics here. E.g. we could agree that all cmt options should start with a `cmt_`. This would allow you to loop over them more easily.

Reply:
Very good idea, I like it

Reply:
I'm not opposed to this in principle, but wouldn't it have been more relevant for #457? Also these options don't always go through CMT right? Though I guess the vast majority of time they will.